core/commands/validate.go

```diff
@@ -0,0 +1,25 @@
+package commands
+
+import (
+	"github.com/spf13/cobra"
+)
+
+// Date Command
+func ValidateCommand() (*cobra.Command, error) {
```

Comment:
I like idea of command that people can add and build based on their own root command. Trick is that this command would probably be root command on it's own. Having validate command accepting your root command will be dependency cycle :)